### PR TITLE
Release 3.2.1

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 3.2.1 (2019-09-27)
+
+- Fixed the grid component to not allow for reordering a column to the left of a locked column. [#72](https://github.com/blackbaud/skyux-grids/pull/72)
+- Fixed the grid component to not allow for reordering a locked column via the sort indicator. [#71](https://github.com/blackbaud/skyux-grids/pull/71)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: I think the `to not allow` wording is hard to follow. I'd reword all three of these items to avoid it. For this one, I'd go with:
```Fixed the grid component to prevent users from reordering locked columns with the sort indicator.```
But if this can be done programmatically as well as by end users, I'd go with:
```Fixed the grid component to prevent the reordering locked columns with the sort indicator.```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+- Fixed the grid component to not scroll when reordering columns on mobile devices. [#70](https://github.com/blackbaud/skyux-grids/pull/70)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: I think the `to not allow` wording is hard to follow. I'd reword all three of these items to avoid it. For this one, I'd go with:
```Fixed the grid component to prevent scrolling when users reorder columns on mobile devices.```
But if this can be done programmatically as well as by end users, I'd go with:
```Fixed the grid component to prevent scrolling during the reordering of columns on mobile devices.```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+# 3.2.1 (2019-09-27)
+
+- Fixed the grid component to not allow for reordering a column to the left of a locked column. [#72](https://github.com/blackbaud/skyux-grids/pull/72)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: I think the `to not allow` wording is hard to follow. I'd reword all three of these items to avoid it. For this one, I'd go with:
```Fixed the grid component to control the reordering of columns around locked columns. You can no longer reorder the columns on the left side of a locked column, and you can no longer move columns from the right side of a locked column to the left side.```
This is quite a bit longer, but I think it more accurately describes the change based on our Slack conversation.

File: CHANGELOG.md (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: The first one is fine

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: The first one is fine

File: CHANGELOG.md (hunk 3 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: That is fine.
